tools/pv_tree_generator/stat_config.proto

```diff
@@ -44,6 +52,7 @@ message PopObsSpec {
   optional string name = 6;
   // Verticals of this spec.
   repeated string vertical = 7;
+  repeated ObsProp obs_props = 8;
```

Comment:
With this, can remove line 42 and 44?

This also means we can treat the current "count", "median age", "median income" in a more general way as they are the same population with different ObsProp.

Reply:
kept it for supporting the old specs, otherwise we would need to revise all the old spec text protos

Reply:
removed mprop and stats_type, revised the proto text